ansible/playbooks/roles/kubernetes_master/templates/kubeadm-config.yml.j2

```diff
@@ -61,3 +61,5 @@ kind: KubeletConfiguration
 apiVersion: kubelet.config.k8s.io/v1beta1
 cgroupDriver: systemd
 rotateCertificates: true
+#enableControllerAttachDetach: {{ specification.extensions.k8s_worker_nodes_capabilities.enable_controller_attach_detach }}
```

Comment:
Adding comment with single `#` sign (or with double `##`) in j2 template will not make the line comment -> it will try to render that line, and access undefined variable which will result in playbook failure "AnsibleUndefinedVariable: 'dict object' has no attribute 'extensions'".

In order to comment line in j2 template you have to use `{# statement #}` syntax.